modules/dynamic-plugins/ref-rh-supported-plugins.adoc

```diff
@@ -36,13 +36,13 @@
 `ARGOCD_AUTH_TOKEN2`
 
 
-|GitHub  |`https://npmjs.com/package/@backstage/plugin-catalog-backend-module-github/v/0.7.8[@backstage/plugin-catalog-backend-module-github]` |0.7.8 
+|GitHub  |`https://npmjs.com/package/@backstage/plugin-catalog-backend-module-github/v/0.7.9[@backstage/plugin-catalog-backend-module-github]` |0.7.9 
 |`./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-dynamic`
 
 `GITHUB_ORG`
 
 
-|GitHub Org  |`https://npmjs.com/package/@backstage/plugin-catalog-backend-module-github-org/v/0.3.5[@backstage/plugin-catalog-backend-module-github-org]` |0.3.5 
+|GitHub Org  |`https://npmjs.com/package/@backstage/plugin-catalog-backend-module-github-org/v/0.3.6[@backstage/plugin-catalog-backend-module-github-org]` |0.3.6 
```

Comment:
Similar comment to above. 
[v0.3.3 corresponds to BS v1.32.6](https://github.com/backstage/backstage/blob/v1.32.6/plugins/catalog-backend-module-github-org/package.json#L3)